assemblyscript-language-plugin.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="UTF-8">
+<title>assemblyscript - iodide</title>
+<link rel="stylesheet" type="text/css" href="https://iodide.io/stable/iodide.stable.css">
+</head>
+<body>
+<script id="jsmd" type="text/jsmd">
+%% meta
+{
+  "title": "assemblyscript",
+  "languages": {
+    "js": {
+      "pluginType": "language",
+      "languageId": "js",
+      "displayName": "Javascript",
+      "codeMirrorMode": "javascript",
+      "module": "window",
+      "evaluator": "eval",
+      "keybinding": "j",
+      "url": ""
+    },
+    "assemblyscript": {
+      "languageId": "assemblyscript",
+      "displayName": "assemblyscript",
+      "codeMirrorMode": "assemblyscript",
+      "keybinding": "a",
+      "url": "https://cdn.jsdelivr.net/npm/assemblyscript@0.3.0/dist/assemblyscript.js",
```

Comment:
`npm` package is deprecated. Use `github:AssemblyScript/assemblyscript`.

Reply:
thanks! 

I'm trying to get a basic example working using the up-to-date versions of binaryen.js, assemblyscript.js, and asc.js. Following the example on the "Using the compiler" wiki page, i thought this would work, but the binary contains no output.

```html
<script src="https://cdn.jsdelivr.net/gh/AssemblyScript/binaryen.js@33313b177292123647c1f773b59f27c863af0c29/index.js"></script>
<script src="https://cdn.jsdelivr.net/gh/AssemblyScript/assemblyscript@086d96b299970ea5b9896ee8a5d5884f0f107f21/dist/assemblyscript.js"></script>
<script src="https://cdn.jsdelivr.net/gh/AssemblyScript/assemblyscript@086d96b299970ea5b9896ee8a5d5884f0f107f21/dist/asc.js"></script>
<script>
const { binary, text, stdout, stderr } = asc.compileString(`export function add(x: i32, y: i32): i32 {
  return x + y;
}`,{optimize: 2})

var compiled = new WebAssembly.Module(binary);
var instance = new WebAssembly.Instance(compiled, {});
console.log(instance.exports.add(41, 1));
</script>
```

I'm not sure if this is related, but I'm also getting an error when I try to build and run the AssemblyScript example in WebAssembly Studio:

`[error]: TypeError: WebAssembly Instantiation: Import #1 module="main" error: module is not an object or function`